src/lib/php/Db/ModernDbManager.php

```diff
@@ -25,8 +25,6 @@ function __construct(Logger $logger)
   public function prepare($statementName, $sqlStatement)
   {
     if (array_key_exists($statementName, $this->preparedStatements)) {
-      echo $this->preparedStatements[$statementName] . "\n";
-      echo $sqlStatement;
```

Comment:
Why did you remove this code ??

Reply:
I was thinking that I'm the one who added them for debugging purpose, let me take them back.